cmd/gateway/hdfs/gateway-hdfs.go

```diff
@@ -507,12 +507,29 @@ func (n *hdfsObjects) GetObject(ctx context.Context, bucket, key string, startOf
 	return hdfsToObjectErr(ctx, err, bucket, key)
 }
 
+func (n *hdfsObjects) isObjectDir(bucket, object string) bool {
```

Comment:
Is `isObjectEmptyDir` a better name?

Reply:
No the idea is that it's a dir only if it is empty so calling it empty dir is not useful.